packages/paste-core/components/checkbox/src/Checkbox.tsx

```diff
@@ -55,14 +55,7 @@ const CheckboxIcon: React.FC<{
   disabled: boolean | undefined;
   checked: boolean | undefined;
 }> = ({checked, disabled, indeterminate}) => {
-  let color = 'transparent' as TextColorOptions;
```

Comment:
Hmm I wonder why this ever was an option?

Reply:
It was my bad from the initial dark mode PR that carried over here.